FlorenceBML/BMLacq776.xml

```diff
@@ -30,10 +31,731 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                </msItem>
+                                <msItem xml:id="ms_i1.11.4">
+                                    <locus from="101vb" to="102ra"/>
+                                    <title ref="LIT1295Dersan#SalamMaskaram"/>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="ms_i1.12">
+                                <title ref="LIT1295Dersan#Teqemt"/>
+                                <msItem xml:id="ms_i1.12.1">
+                                    <locus from="102ra" to="105va"/>
+                                    <title ref="LIT1295Dersan#HomilyTeqemt"/>
+                                    <incipit xml:lang="gez"><hi rend="rubric">በስመአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ ድርሳን፡</hi> ዘቅዱስ፡ 
+                                        <gap reason="omitted" unit="chars" quantity="3"/>ሊቀ፡ መላእክት፡ ዘይትነበብ፡ አመ፡ ፲ወ፪ለጥቅምት፡ ትንብልናሁ፡ ምስለ፡ ገብሩ፡ 
+                                        <gap reason="omitted" unit="chars" quantity="3"/> ሰምዑ፡ ሕዝበ፡ ወንጌል፡ ነፍስ፡ ዘኢኃሠሠት፡ ዋሕሠ፡ በህይወታ፡ ኢተረክብ፡ ዋህሰ፡ 
+                                        በጊዜ፡ መታ፡ ወከመዝ፡ ኮነ፡ ፳<cb n="102rb"/>ብእሲ፡ ዘቦቱ፡ ሸዳ፡ ወኃጥአ፡ ዘይፈዪ፡ ኃሣሥዎ፡ አኃዝዎ፡ ዘወሀብዎ፡ 
+                                        <add place="above">ኒ የ</add>ወበማዕከለ፡ አውድ፡ ወኃጥአ፡ ዘይፈድዮሙ፡ </incipit>
+                                    <explicit xml:lang="gez">ንዑ፡ ኃቤየ፡ እኁበ፡ ክብረ፡ ወምንተ፡ ትሂል<pb n="105va"/>ዉ፡ ምስሌየ፡ በትፍሥሕት፡ ወበሕማት፡ ወኢይሂሱ፡ 
+                                        ቃልየ፡ በረከቱ፡ ወኃብተ፡ ለቅዱስ፡ <hi rend="rubric">ሚካኤል፡ ሊቀ፡ መላእክት፡</hi> ወበረከታ፡ 
+                                        ለእ<hi rend="ligature">ግዚ</hi>እትነ፡ <hi rend="rubric">ማርያም፡ ወላዲተ፡ አምላክ፡</hi> በረከተ፡ ነቢያት፡ ወሐዋርያት፡ ጻድቃን፡ 
+                                        ወሰምዕት፡ የሃሉ፡ ምስለ፡ ገብሩ፡
+                                        <hi rend="rubric"><persName ref="PRS14011GabraMikael" role="owner">ገብሚካኤል፡ </persName></hi>
+                                            </explicit>
+                                </msItem>
+                                <msItem xml:id="ms_i1.12.2">
+                                    <locus from="105va" to="106ra"/>
+                                    <title>Miracle of the rich man who recovers his possessions.</title>
+                                </msItem>
+                                <msItem xml:id="ms_i1.12.3">
+                                    <locus from="106ra" to="107ra"/>
+                                    <title ref="LIT1295Dersan#CommemorativeTeqemt"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.12.4">
+                                    <locus target="#107r"/>
+                                    <title ref="LIT1295Dersan#SalamTeqemt"/>
+                                </msItem>
+                            </msItem>
+                            <note>Spaces left free for owner names throughout except for <locus target="#52rb #105va #106ra"/>, where the name of the owner
```

Comment:
This information can to an "extra"

Reply:
I encoded it as an ownership note even though these are non existing ownership notes.